tools/UpdateBuildMachineGitInstalls.ps1

```diff
@@ -0,0 +1,17 @@
+[CmdLetBinding()]
+Param(
+    )
+
+Import-Module -Name $PSScriptRoot\BuildMachineUtils.psm1 -DisableNameChecking
+
+$adalPath = Get-AdalPath
+
+Write-Verbose "ADALPath: $adalPath"
+Add-Type -Path "$adalPath\lib\net45\Microsoft.IdentityModel.Clients.ActiveDirectory.dll"
+
+$result = Queue-BuildOnMachine -MachineName "PKGESWINUI24" -ClientAlias "winui2" -BuildId "17093" # Install GVFS
```

Comment:
Do you need to filter to ClientAlias? I wouldn't since you already have MachineName.

Reply:
Good point, I don't believe we do.